libs/tokens/src/pure/TokenLogo/index.tsx

```diff
@@ -63,17 +64,23 @@ export const TokenLogoWrapper = styled.div<{ size?: number; sizeMobile?: number
 `
 
 const ChainLogoWrapper = styled.div<{ size?: number }>`
-  width: ${({ size = 16 }) => size}px;
-  height: ${({ size = 16 }) => size}px;
-  display: flex;
-  align-items: center;
-  justify-content: center;
-  border-radius: 50%;
-  border: 1px solid white;
-  position: absolute;
-  padding: 0;
-  bottom: -2px;
-  right: -2px;
+  ${({ size = 16 }) => {
+    const getBorderWidth = () => Math.max(1.5, Math.min(2.2, size * 0.15))
```

Comment:
It's better to move `getBorderWidth` out of the styles callback.
Each time the callback is called it will create a new instance of the function.
Not a big deal, but still a performance issue.

Reply:
Addressed now. Also added more const's on top with more descriptive names.